### PR TITLE
JS Login Logic + CSS Tweaks

### PR DESCRIPTION
**Pull request by mihneat**

### CODE REVIEW

**Comment by robpan38** [comment]
[LOGIN SCRIPT](https://trello.com/c/aikXXilk/5-login-script)

File: scripts/loginLogic.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+let loginInput = document.querySelector(".login-input-field");
+let loginBtn = document.querySelector(".login-btn");
+
+loginInput.addEventListener("input", () => {
+    debugger;
```
Comment (anbalase) [inline scripts/loginLogic.js]: oops

Comment (mihneat) [reply scripts/loginLogic.js]: fixed :)

**Comment by alexdudescu** [comment]
We should either add something to this commit or drop it:  https://github.com/robpan38/pathcord/pull/3/commits/92aecf9eaa486e68f1692e4def6c7486e99c89ad

**Comment by mihneat** [comment]
I vote drop

**Comment by robpan38** [comment]
consider that commit gone 😎

File: scripts/loginLogic.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+loginInput.addEventListener("input", () => {
+    if (loginInput.value === "" && loginBtn.classList.contains("active-login-btn")) {
+        loginBtn.classList.remove("active-login-btn");
+    } else if (loginInput.value !== "" && !loginBtn.classList.contains("active-login-btn")) {
+        loginBtn.classList.add("active-login-btn");
+    }
+});
+
+loginBtn.addEventListener("click", () => {
+
```
Comment (alexdudescu) [inline scripts/loginLogic.js]:
```suggestion
```

Comment (robpan38) [reply scripts/loginLogic.js]: multumim!

**Review by alexdudescu — APPROVED** [review]
🎉

**Pull request merged**